tools/tasks/tslint.ts

```diff
@@ -11,7 +11,9 @@ export = function tslint(gulp, plugins) {
     ];
 
     return gulp.src(src)
-      .pipe(plugins.tslint())
+      .pipe(plugins.tslint({
+        rulesDirectory: NG2LINT_RULES
```

Comment:
Yes, I intended to do so but I didn't manage to make it work. Not sure if gulp-tslint recognizes this option.